community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/StoreVersionCheck.java

```diff
@@ -83,7 +83,7 @@ private String typeDescriptor( String expectedVersion )
         return expectedVersion.substring( 0, spaceIndex );
     }
 
-    private String readVersion( PagedFile pagedFile, String expectedVersion ) throws IOException
+    public static String readVersion( PagedFile pagedFile, String expectedVersion ) throws IOException
```

Comment:
With 2 new static methods `StoreVersionCheck` became more utility class then version checker, can we consider split it into 2 separate classes instead?